src/parser.ts

```diff
@@ -19,6 +19,9 @@ export const createDictccUrl = ({
   return url.href
 }
 
+export const getDictccAudioUrl = (translationId: number) =>
+  `https://audio.dict.cc/speak.audio.v2.php?type=mp3&id=${translationId}&lang=de_rec_ip`
+
 export const getHtmlString = async (url: string) => (await fetch(url)).text()
```

Comment:
`getDictccAudioUrl` is unused, isn't it?

Reply:
Good catch, yes we can remove it